### ISSUE TITLE
Project generation fails due to missing files

### ISSUE THREAD

**Issue by svallory** [langium/langium#1037]
Langium version: -
Package name: generator-langium

<img width="562" alt="image" src="https://user-images.githubusercontent.com/117560/235775041-aa426060-5af2-430a-87bd-214ef49042a4.png">

## Steps To Reproduce

1. Just try to create a new project and answer Y to all optional generation questions

<img width="631" alt="image" src="https://user-images.githubusercontent.com/117560/235775297-01c06e6f-bf3a-472f-ae83-b354ef257eac.png">

## The current behavior

```
✖ An error occured while running langium:app#writing
Error langium

Trying to copy from a source that does not exist: 
/Users/svallory/.asdf/installs/nodejs/18.12.1/lib/node_modules/generator-langium/langium-template-core
```

## The expected behavior

New project is generated

**Comment by msujew** [comment]
Oh, well, we just refactored our project generator and of course forgot to add the files into the published npm package. If you're new to Langium, you can just downgrade to `generator-langium@1.1.0`. We'll release a new generator version tomorrow that fixes the issue.

**Comment by msujew** [comment]
The fix has been merged and `generator-langium@1.2.1` has been released.
